servicefabric/2016-01-28/swagger/servicefabric.json

```diff
@@ -3055,6 +3499,30 @@
     }
   },
   "definitions": {
+    "RegisterClusterPackage": {
+      "type": "object",
+      "description": "The package of the register cluster",
+      "properties": {
+        "CodeFilePath": {
```

Comment:
Are you sure that the properties on the wire are actually PascalCase? Usually the convention is camelCase, hence wanted to confirm.